python/hidet/driver.py

```diff
@@ -278,13 +278,14 @@ def build_ir_module_batch(
         ]
         build_results = []
         if parallel:
+            num_jobs, mem_for_worker = option.get_parallel_tune()
+            num_jobs = os.cpu_count() if num_jobs == -1 else num_jobs
+            mem_for_worker *= 1024**3
             # Set the affinity of current process. Some package such as numpy will change affinity of current process,
             # which might limit the parallelism of compilation.
-            os.sched_setaffinity(0, range(os.cpu_count()))
+            os.sched_setaffinity(0, range(num_jobs))
```

Comment:
Hi Xin, it's better to leave this to `os.cpu_count()`. Basically, this sets the visible vcpu to our process.